twine/utils.py

```diff
@@ -75,17 +75,17 @@ def get_config(path: str) -> Dict[str, RepositoryConfig]:
     config: DefaultDict[str, RepositoryConfig]
     config = collections.defaultdict(lambda: defaults.copy())
 
-    index_servers = parser.get(
-        "distutils", "index-servers", fallback="pypi testpypi"
```

Comment:
This fallback was adding `testpypi` to every config file that didn't have `distutils` section with `index-servers` option, thus the tests included extra values.

Reply:
That fallback is very much intentional. Pypi and testpypi are reasonable defaults for the index servers config.

Reply:
Right, and PyPI is the default fallback, which I'd expect.

The existing if condition checks for testpypi and adds it only if specified as an entry.

Otherwise why is that conditional there?

Reply:
Sadly the config file isn't the single source of truth. Someone can specify `--username`, `--password` and `--repository` without including a section for repository in the config file. So someone could upload to testpypi today with this fallback without needing the section to be present.

Reply:
I read through your response a few times, and concluded that the logic we have today that does a get/fallback isn't what you're describing as the desired default, so I've changed it to no longer conditionally apply testpypi, rather to set the defaults (which can be overridden by config file or CLI), which preserves the behavior you're describing, while still removing the extra `distutils` section.